new-examples/directories/directories.metatags

```diff
@@ -0,0 +1,2 @@
+description: This BBE shows how to perform directory operations in Ballerina.
```

Comment:
```suggestion
description: This BBE shows how to perform directory operations in Ballerina.
```
Please change this in all BBEs.

Reply:
Contained the specific description of the BBEs in the description but all BBEs are related to the file module. So these may seem the same description. Can you check them?

Reply:
+1